### ISSUE TITLE
Image browser does not load correctly when content type has not WYSIWYG field

### ISSUE THREAD

**Issue by mazzech** [backdrop/backdrop-issues#4527]
**Description of the bug**

Whenever I have content types without any HTML formatte field, the image browser does not load correctly.

**Steps To Reproduce**

To reproduce the behavior:
- Create a new content type
- Delete the body field
- add an image field
- optional: add some non-WYSIWYG fields
- create the node and try to choose an existing image ("Select from Image Library")

![Screenshot 2020-08-08 at 11 30 27](https://user-images.githubusercontent.com/5985692/89707023-11b4a180-d96b-11ea-9cdb-214a81599dac.png)

- Backdrop Version 1.16.2 
- various browsers on OSX

As soon as any WISYWIG field is added, browser works correctly, shows images in matrix

![Screenshot 2020-08-08 at 11 35 28](https://user-images.githubusercontent.com/5985692/89707112-956e8e00-d96b-11ea-8833-fb60a94552ad.png)

**Comment by indigoxela** [comment]
I can confirm that.

It's not even necessary to delete the body field. Simply setting its format to "Plain text" already causes the problem.

**Comment by indigoxela** [comment]
A [PR](https://github.com/backdrop/backdrop/pull/3225) is available for testing and review.

**Comment by BWPanda** [comment]
Thanks @indigoxela! Left a review requesting changes.

**Comment by klonos** [comment]
Yes, thanks @indigoxela 👍 ...the PR works as advertised; we just need to fix the inline comment and move it somewhere where it makes more sense. Once that's done, this is RTBC.

**Comment by indigoxela** [comment]
Hey @BWPanda and @klonos many thanks for testing and reviewing. Unfortunately your requests re position of the comment are pretty confusing for me.

**Comment by indigoxela** [comment]
@BWPanda many thanks for clarifying.

The PR has been updated:
- @klonos I took the first sentence of your suggestion, but skipped the second. The comment seemed too wordy with it.
- @BWPanda I rearranged lines as requested.

**Comment by BWPanda** [comment]
RTBC from me!

**Comment by mazzech** [comment]
wow, fastest fix ever, thank you all:-)

**Comment by quicksketch** [comment]
Merged https://github.com/backdrop/backdrop/pull/3225 into 1.x and 1.16.x. Thanks everyone!

**Comment by indigoxela** [comment]
Reopening this as it caused a regression: on nodes *with* ckeditor enabled textareas and image fields the editor is broken:

js error:
`Uncaught TypeError: Backdrop.editors is undefined`

**Comment by indigoxela** [comment]
I opened a new Issue for the regression: https://github.com/backdrop/backdrop-issues/issues/4592
